frontend/src/metabase/common/components/QueryColumnPicker/BucketPickerPopover/BaseBucketPickerPopover.styled.tsx

```diff
@@ -6,11 +6,14 @@ import type { ColorName } from "metabase/lib/colors/types";
 
 export const TriggerIcon = styled(Icon)`
   color: ${color("white")} !important;
+  flex: 0 0 auto;
 `;
 
 export const TriggerButton = styled.button`
   display: flex;
   align-items: center;
+  min-width: 0;
```

Comment:
@kamilmielnik shouldn't we use flex-base, flex-shrink? just curious

Reply:
I think we cannot (I tried many options) due to content of the button having ellipsis applied on it.
`flex-shrink` just does nothing (it's `flex: 0 1 auto` by default already).

This `min-width: 0` trick already bailed me out many times in the past.